### PR TITLE
update Docker images to support ES6.0 alpha, removed groovy scripting

### PR DESCRIPTION
**Pull request by p365labs**
these amends will allow to start ES 6.0 properly (mapper-attachments plugin, scripts.inline, groovy scripts has been removed)

### CODE REVIEW

**Comment by p365labs** [comment]
@ruflin with these small amends on the Dockerfile and elasticsearch.yml we will be able to execute tests ... as expected the main effort should be pointed on the Types removal

File: env/elasticsearch/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,6 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.0.0-alpha2
 MAINTAINER Nicolas Ruflin <spam@ruflin.com>
 
-RUN /usr/share/elasticsearch/bin/elasticsearch-plugin install mapper-attachments
```
Comment (ruflin) [inline env/elasticsearch/Dockerfile]: Should we replace this with `ingest-attachment`?

**Comment by ruflin** [comment]
Nice, see my comment.

Unfortuantely travis did not build it as I told travis to only build master as it would otherwise build my branches always twice.

**Comment by p365labs** [comment]
@ruflin I've seen ur comment, sonds good. What if I try to merge it on master? as we can start share errors on TYPES.. and take a decision on how to implement a strategy for the new ES6.0 Indices ?

**Comment by ruflin** [comment]
@p365labs Open a PR directly against master so we see the errors.

**Comment by p365labs** [comment]
ok!

**Pull request closed without merging**